src/awesome_map/BoundaryInterceptor.js

```diff
@@ -365,20 +408,31 @@ define(function(require) {
          *        The difference in dimension between the viewport and content:
          *        for content that fits the viewport, headroom will be positive;
          *        for content that overflows, headroom will be negative.
-         * @param targetValue - The value being bound.
+         * @param {number} targetValue - The value being bound.
+         * @param {boolean} centerContent
+         *        If this is true, and headroom is greater than zero, the returned
+         *        value will force content to be centered in the viewport.
+         * @param {boolean} pinToEdge
+         *        If this is true, and headroom is greater than zero, the returned
+         *        value will force content to be aligned to the left or top
+         *        of the viewport. This setting is ignored if centerContent
+         *        is true.
          * @returns {number}
          * @private
          */
-        _getBoundedValue: function(headroom, targetValue) {
+        _getBoundedValue: function(headroom, targetValue, centerContent, pinToEdge) {
             var boundary = targetValue;
 
             // Content dimension fits within the viewport:
             if (headroom > 0) {
 
                 // If centering, use an absolute position.
-                if (this._centerContent) {
+                if (centerContent) {
```

Comment:
I had to pass it in as a parameter instead of using a property because we now use a different property for bounding X values (_centerContentX) vs. Y values (_centerContentY). In the past we only had a single _centerContent property.